pkg/types/powervs/validation/platform.go

```diff
@@ -0,0 +1,18 @@
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types/powervs"
+)
+
+// ValidatePlatform checks that the specified platform is valid.
+func ValidatePlatform(p *powervs.Platform, fldPath *field.Path) field.ErrorList {
```

Comment:
This is missing validation for most of the fields.

Reply:
yah. this is a wip in our fork: https://github.com/openshift-powervs/installer/pull/102/files
ok to pull that in in a followon PR? for now would like to get these in so we can stop using that fork